src/sentry/utils/performance_issues/grouptype.py

```diff
@@ -0,0 +1,181 @@
+from dataclasses import dataclass
+
+from sentry.issues.grouptype import (
+    GroupCategory,
+    GroupType,
+    NoiseConfig,
+    NotificationConfig,
+    NotificationContextField,
+)
+from sentry.types.group import PriorityLevel
+from sentry.utils.performance_issues.detector_handlers.n_plus_one_api_calls_detector_handler import (
+    NPlusOneAPICallsDetectorHandler,
+)
+
+
+# used as an additional superclass for Performance GroupType defaults
+class PerformanceGroupTypeDefaults:
+    noise_config = NoiseConfig()
+
+
+@dataclass(frozen=True)
+class PerformanceSlowDBQueryGroupType(PerformanceGroupTypeDefaults, GroupType):
```

Comment:
To guarantee these are imported, we need them to be in the root of an app: https://github.com/getsentry/sentry/blob/57077af0a7b2d00366fdc78e0d5eee709eec1a02/src/sentry/issues/grouptype.py#L658-L671

I'm not totally sure where these should go though, since we don't have a performance app or anything...